File: app/src/main/java/eu/kanade/tachiyomi/ui/reader/viewer/pager/vertical/VerticalReader.java
package eu.kanade.tachiyomi.ui.reader.viewer.pager.vertical;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import eu.kanade.tachiyomi.ui.reader.viewer.pager.PagerReader;

public class VerticalReader extends PagerReader {

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedState) {
        VerticalPager pager = new VerticalPager(getActivity());
        initializePager(pager);
        return pager;
    }

}

File: app/src/main/java/eu/kanade/tachiyomi/ui/reader/viewer/pager/vertical/VerticalReader.kt
package eu.kanade.tachiyomi.ui.reader.viewer.pager.vertical

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import eu.kanade.tachiyomi.ui.reader.viewer.pager.PagerReader

/**
 * Vertical reader.
 */
class VerticalReader : PagerReader() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedState: Bundle?): View? {
        return VerticalPager(activity).apply { initializePager(this) }
    }

}
